lib/rototiller/task/params/option.rb

```diff
@@ -40,6 +40,11 @@ def add_argument(*args, &block)
       def to_str
         [@name.to_s, @arguments.to_s].compact.join(' ')
       end
+
+      def stop
+        return true if @arguments.stop?
+        return true unless @name
+      end
```

Comment:
i guess just because we want to stop if they don't receive a name? 
this works (doesn't stop) with `@name = ''`, yeah?